src/common/TweetBtn.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function TweetBtn(props) {
+    return <button className={"tweet-btn ".concat(props.size)} type="button" onClick={(e) => props.onClick()}>
```

Comment:
Instead of the .concat you can use 
1. https://www.npmjs.com/package/classnames
2. Just use className={"tweet-btn " + props.size}
3. className={`tweet-btn ${props.size}`} - I prefer this one